draft-ietf-quic-multipath.md

```diff
@@ -351,31 +314,40 @@ does not discuss when a client decides to initiate a new path. We
 delegate such discussion in separate documents.
 
 To let the peer open a new path, an endpoint needs to provide its peer with connection IDs
-and associated Path Identifiers for the new path.
+for at least one unused path identifier. Endpoints SHOULD use the MP_NEW_CONNECTION_ID
+frame to provide new connection IDs and, respectively, the MP_RETIRE_CONNECTION_ID frame to
+retire connection IDs after a successful handshake indicating multipath support by both endpoints.
+
+Endpoints MUST NOT issue connection IDs with path identifiers larger than
+the path limitation declared by the initial_max_paths transport parameter
+or MAX_PATHS frames.
```

Comment:
I think this is a bit ambiguous here, as the current text does not specify whether it refers to the value advertised by the sender, the receiver, or the minimum of both. To me, we should clarify this is relative to the value advertised by the peer.

```suggestion
Endpoints MUST NOT issue connection IDs with path identifiers larger than
the path limitation advertised by the peer, corresponding to the maximum value
between the peer's initial_max_paths transport parameter and received
MAX_PATHS frames.
```

Also, I would expect discussing here how the receiver should react if it receives Connection IDs for non-authorised paths IDs. Closing the connection with some MP_PROTOCOL_VIOLATION error?